projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -1,6 +1,9 @@
 import { Injector, Renderer2 } from '@angular/core';
-import { TimeRange } from '@hypertrace/common';
-import { ContainerElement, mouse, select } from 'd3-selection';
+
+import { TimeRange, TimeRangeService } from '@hypertrace/common';
+import { BrushBehavior, brushX, D3BrushEvent } from 'd3-brush';
+// tslint:disable
```

Comment:
```suggestion
// tslint:disable-next-line: no-restricted-globals
```

Current version turns off tslint for the whole file